 Analyzing entries 0 - 13, now 0 , sum entries = 13050 . . . Used time: 0s Expected left time: 0s          
 Analyzing entries 0 - 13, now 1 , sum entries = 13050 . . . Used time: 0s Expected left time: 0s          
 Analyzing entries 0 - 13, now 2 , sum entries = 13050 . . . Used time: 0s Expected left time: 0s          
 Analyzing entries 0 - 13, now 3 , sum entries = 13050 . . . Used time: 0s Expected left time: 0s          
 Analyzing entries 0 - 13, now 4 , sum entries = 13050 . . . Used time: 0s Expected left time: 0s          
 Analyzing entries 0 - 13, now 5 , sum entries = 13050 . . . Used time: 0s Expected left time: 0s          
 Analyzing entries 0 - 13, now 6 , sum entries = 13050 . . . Used time: 0s Expected left time: 0s          
 Analyzing entries 0 - 13, now 7 , sum entries = 13050 . . . Used time: 0s Expected left time: 0s          
 Analyzing entries 0 - 13, now 8 , sum entries = 13050 . . . Used time: 0s Expected left time: 0s          
 Analyzing entries 0 - 13, now 9 , sum entries = 13050 . . . Used time: 0s Expected left time: 0s          
 Analyzing entries 0 - 13, now 10 , sum entries = 13050 . . . Used time: 0s Expected left time: 0s          
 Analyzing entries 0 - 13, now 11 , sum entries = 13050 . . . Used time: 0s Expected left time: 0s          
 Analyzing entries 0 - 13, now 12 , sum entries = 13050 . . . Used time: 0s Expected left time: 0s          
terminate called after throwing an instance of 'std::out_of_range'
  what():  vector::_M_range_check: __n (which is 0) >= this->size() (which is 0)
